src/ProjectTemplates/Web.ItemTemplates/content/RazorComponent/.template.config/template.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "ASP.NET"
+  ],
+  "name": "Razor Component",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A Razor Component",
```

Comment:
To make this more descriptive, how about: A reusable web UI component implemented with Razor.

Reply:
You super sure about putting the word `web` so close to the word `component`?